supply-chain-security/compromises/2022/php-pear-compromise.md

```diff
@@ -0,0 +1,42 @@
+<!-- cspell:ignore CVE -->
+<!-- cspell:ignore PEAR -->
+<!-- cspell:ignore PHP -->
+<!-- cspell:ignore TAR -->
+<!-- cspell:ignore unpacthed -->
+<!-- cspell:ignore superseded -->
+<!-- cspell:ignore Packagist -->
+<!-- cspell:ignore pearweb -->
+<!-- cspell:ignore archives -->
+# PEAR PHP Package Manager compromise
+
+A case of flawed password reset logic and unpacthed known CVE of
+a third-party dependency allowed PEAR's package manager compromise.
+
+PEAR is a PHP package manger that has been superseded by Packagist,
+but still a relevant part of the PHP developer ecosystem.
+
+A code security flaw found in the source code of the package manager's
+password reset logic, along with the detection of an unpacthed publicly
+known vulnerable third-party dependency to manage TAR file archive
+allowed attackers to compromise PEAR's package manager registry and
+gain access to the server.
```

Comment:
I feel there are two issues here and we'd like to make sure they are understood separately:

1. There was a pwreset flaw that allowed access to attackers
2. This, combined with a vulnerable tar application, allowed attackers to overwrite exisitng  packages in pearweb

I feel lines 12 and 13 try to make that argument, but perhaps we could move them to an itemized list just to make it easier to grok?

Reply:
Yep, agree. Would be good to call it out in a bullet where it's clearer.

Reply:
Updated